src/display/fetch_stream.js

```diff
@@ -248,10 +248,7 @@ class PDFFetchStreamRangeReader {
         this._reader = response.body.getReader();
       })
       .catch(reason => {
```

Comment:
Does it work the same if you simply do `.catch(this._readCapability.reject);` instead here?
If so, please update the patch to fix this :-)

Reply:
Yes, that works too. Done via 6687f2c3a05c097782fb261b6ef039c33a993ca0.

Reply:
Please note that we don't allow separate fixup commits, and that this change needs to be folded into its parent commit instead.

Also, you can please remove the "[misc]" part of the commit message since it seems unnecessary.

Reply:
We usually squash merge PRs with verbose commit messages and keep the log on the PR as is :slightly_smiling_face:
I've squashed the changes my self now and force-pushed the new commit.